annotations/openapi/oas-extension.raml

```diff
@@ -0,0 +1,8 @@
+#%RAML 1.0 AnnotationTypeDeclaration
+
+usage: |
+  This annotation represents an equivalent to the `^x-` property which 
+  is used to define extensions for the OpenAPI Specification. 
+
+type: any
+displayName: Extensions (OAS)
```

Comment:
As for me it does not looks like a good idea to convert all extensions into 1 annotation. (Imagine round trip from RAML to OAS and back, ideally we should convert raml annotations to extensions and then back to same annotations)